lib/ansible/modules/network/nxos/nxos_nxapi.py

```diff
@@ -165,6 +203,20 @@ def map_obj_to_commands(want, have, module):
     send_commands = list()
     commands = dict()
 
+    device_info = get_capabilities(module).get('device_info')
```

Comment:
`get_capabilities` is used in `check_args` as well. Instead of calling the function twice, please call the function in `main()` as pass that as argument to the methods.